core/templates/pages/volunteer-page/volunteer-page.component.html

```diff
@@ -270,14 +270,13 @@ <h4 class="carousel-sub-header">{{ 'I18N_VOLUNTEER_PAGE_VOLUNTEER_SECTION_LESSON
 
     <oppia-primary-button
         buttonHref="https://forms.gle/rhFYoLLSFr3JEZHy8"
-        [customClasses]="['btn']"
         buttonText="{{ 'I18N_VOLUNTEER_PAGE_SECTION_ONE_BUTTON' | translate }}">
     </oppia-primary-button>
   </div>
 </div>
 
 <style>
-  .volunteer-page button {
+  .volunteer-page a {
```

Comment:
Shouldn't these classes be passed through the button's customClasses?

Reply:
Done